examples/meta/generator/parse.py

```diff
@@ -85,7 +77,8 @@ def addShogunTypes(self, dictionary):
     t_EQUALS = "="
     t_LPAREN = "\("
     t_RPAREN = "\)"
-    t_COMMENT = r"\#.*\n"
```

Comment:
There was a bug with the parsing error messages not counting comments as a line, so I've moved this rule into one with the side effect of incrementing the line count like the rule for the NEWLINE token.